packages/front-generator/src/generators/react-typescript/entity-cards/template/EntityCards.tsx.ejs

```diff
@@ -1,24 +1,62 @@
 import * as React from "react";
 import {observer} from "mobx-react";
+
 import {<%= entity.className %>} from "<%= relDirShift %>cuba/entities/<%= entity.name %>";
-import {Card, Icon} from "antd";
-import {collection} from "@cuba-platform/react-core";
-import {EntityProperty} from "@cuba-platform/react-ui";
+import {Card} from "antd";
+import {collection, MainStoreInjected, injectMainStore} from "@cuba-platform/react-core";
+import {
+addPagingParams, createPagingConfig,
+defaultPagingConfig,
+EntityProperty,
+Paging,
+setPagination,
+Spinner
+} from "@cuba-platform/react-ui";
+import {action, IReactionDisposer, observable, reaction} from "mobx";
+import {PaginationConfig} from "antd/es/pagination";
+import {RouteComponentProps} from "react-router";
+
+type Props = MainStoreInjected & RouteComponentProps;
 
+@injectMainStore
 @observer
-export class <%= className %> extends React.Component {
+export class <%= className %> extends React.Component<Props> {
+
+  dataCollection = collection<<%= entity.className %>>(<%= entity.className %>.NAME, {
+    view: '<%=view.name%>',
+    sort: '-updateTs',
+    loadImmediately: false
+  });
 
-  dataCollection = collection<<%= entity.className %>>(<%= entity.className %>.NAME, {view: '<%=view.name%>', sort: '-updateTs'});
+  @observable paginationConfig: PaginationConfig = { ...defaultPagingConfig };
+  reactionDisposer: IReactionDisposer;
   fields = [<% attributes.forEach(p => { %>
```

Comment:
why not use `-%>` as on `List` template (to avoid additional lines between fields)

Reply:
Fixed.